sample.java

```
@@ -0,0 +1,31 @@
+package exceptionhandling2;
+
+
+
+public class sample {
+public void mth1() {
+	mth2();
+	System.out.println("caller");
+}
+public void mth2() {
+	
+		try {
+			mth2();
```

Need to throw an exception